src/main/resources/org/kaazing/eclipse/KaazingJavaFormatting.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="12">
+<profile kind="CodeFormatterProfile" name="Kaazing" version="12">
```

Comment:
Recommend renaming this file from `KaazingJavaFormatting.xml` to `CodeFormatterProfile.xml`.